app/models/vm_or_template.rb

```diff
@@ -1884,6 +1884,10 @@ def supports_snapshots?
     false
   end
 
+  def supports_provisioning?
+    true
```

Comment:
@gmcculloug i am trying to use the terminology of method name that we use in other places in UI to figure out if a button should be available or unavailable to perform a task.